examples/pos-mainchain/config/default/genesis_assets.json

```diff
@@ -158,14 +113,14 @@
 													"fieldNumber": 1
 												},
 												"size": {
-													"dataType": "uint32",
```

Comment:
so do you want to order all params? (Like `fieldNumber` first, `dataType` second)